python/requirements.txt

```diff
@@ -1,6 +1,5 @@
 Cython>=0.22
-cmdstanpy==0.9.77
-pystan~=2.19.1.1
```

Comment:
If we're removing this should we catch import errors in parts of the code which use pystan? With a check for the version as well. Otherwise happy to leave this in for now and do that in another PR.

Reply:
There are a few references to pystan in `setup.py` and `models.py`

They appear straightforward to remove. This would mean that not only is `cmdstanpy` the default backend, but also that `pystan` will not be supported at all

Reply:
I can remove PyStan all together, but the main reason the requirements changed here is so I could do local testing and it snuck into a commit. 

I think it would be worth a separate PR to remove pystan, as it can also do a lot of cleanup/simplifying of the code